packages/types/src/documents/global/config.ts

```diff
@@ -45,7 +47,7 @@ export interface OIDCConfiguration {
   callbackURL: string
 }
 
-export interface OIDCInnerCfg {
```

Comment:
Good improvement, always better to be verbose in type names!

Reply:
As long as we don't get the to point of `OIDCStrategyFactoryProducerWrappingBean` I agree 😂